subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt

```diff
@@ -350,60 +317,6 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
         }
     }
 
-    @Test
-    fun `report line warning on runtime failure in currently edited script when location aware hints are enabled`() {
```

Comment:
:question: this test is about a report for something that is caused by the current script, why is this test removed?